app/src/androidTest/java/org/simple/clinic/bp/BloodPressureHistoryListItemDataSourceTest.kt

```diff
@@ -41,6 +45,9 @@ class BloodPressureHistoryListItemDataSourceTest {
   @field:[Inject Named("time_for_measurement_history")]
   lateinit var timeFormatter: DateTimeFormatter
 
+  @get:Rule
+  val ruleChain: RuleChain = Rules.global()
```

Comment:
As the database is getting cleared now, does the Quarantine rule (`Rules.global()`) still need to be used?

Reply:
That needs to be added for all instrumented tests.

Reply:
The point of the rule is to continuously observe all tests for flakiness, which could get introduced in the future.

Reply:
Oh gotcha